packages/block-library/src/audio/edit.native.js

```diff
@@ -97,6 +112,9 @@ function AudioEdit( {
 	}
 
 	function getBlockControls( open ) {
+		if ( isCaptionSelected ) {
```

Comment:
could we move this logic to the `getBlockUI` function? My idea is: 

> { ! isCaptionSelected && getBlockControls( open ) }

I have seen this pattern throughout the codebase when a boolean property needs to be checked to determine if a component should be rendered or not. Let me know what you think.

Reply:
Makes sense 👍